src/cljs/main/broadfcui/main.cljs

```diff
@@ -205,55 +243,56 @@
                                (contains? (:user-status @state) :signed-in))]
        [:div {}
         (when (:config-loaded? @state)
-          [StatusAlertContainer])
+          [BannerContainer])
         (when (and (contains? user-status :signed-in) (contains? user-status :refresh-token-saved))
           [auth/RefreshCredentials {:auth2 auth2}])
-        [:div {:style {:backgroundColor "white" :padding 20}}
-         (when-not (contains? user-status :signed-in)
-           (style/render-text-logo))
-         [:div {}
-          (when auth2
-            [auth/LoggedOut {:auth2 auth2 :hidden? sign-in-hidden?
-                             :on-change (fn [signed-in? token-saved?]
-                                          (swap! state update :user-status
-                                                 #(-> %
-                                                      ((if signed-in? conj disj)
-                                                       :signed-in)
-                                                      ((if token-saved? conj disj)
-                                                       :refresh-token-saved))))}])
+        [:div {:style {:position "relative"}}
```

Comment:
Again, just checking, do we need to create a new positioning context again here?